src/applications/personalization/profile/actions/directDeposit.js

```diff
@@ -98,16 +98,56 @@ export function saveDirectDeposit(
 
     const response = await getData(DIRECT_DEPOSIT_API_ENDPOINT, {
       method: 'PUT',
+      headers: {
+        'Content-Type': 'application/json',
+      },
```

Comment:
@tpharrison here is where I'm fixing my PUT request headers. They will merge with the default headers of the defaultSettings:

```
const defaultSettings = {
    method: 'GET',
    credentials: 'include',
    headers: {
      'X-Key-Inflection': 'camel',
      'Source-App-Name': window.appName,
      'X-CSRF-Token': csrfTokenStored,
    },
  };
  ```